packages/nouns-contracts/contracts/governance/NounsDAOAdmin.sol

```diff
@@ -528,7 +541,11 @@ library NounsDAOAdmin {
      * @param timelockV1 the new timelockV1 contract
      * @param admin the new admin address
      */
-    function _setTimelocksAndAdmin(address timelock, address timelockV1, address admin) external onlyAdmin {
```

Comment:
@eladmallel looks like your formatter is misconfigured.
this line is < 120 chars